modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -155,10 +149,11 @@ export type GetInvoicesResponse = t.TypeOf<typeof GetInvoicesResponse>;
 
 export interface ILightning {
   createInvoice(params?: CreateInvoiceParams): Promise<CreateInvoiceResponse>;
-  createDepositAddress(params?: CreateDepositAddressParams): Promise<CreateDepositAddressResponse>;
+  createDepositAddress(): Promise<CreateDepositAddressResponse>;
   payInvoice(params?: PayInvoiceParams): Promise<PayInvoiceResponse>;
   getBalance(): Promise<GetBalanceResponse>;
   withdraw(params?: LightningWithdrawalParams): Promise<WithdrawResponse>;
   deposit(params?: LightningDepositParams): Promise<DepositResponse>;
-  getInvoices(params?: GetInvoicesParams): Promise<GetInvoicesResponse>;
+  // Implemented in lnd-middleware but not in wallet-platform
+  getInvoices(): Promise<GetInvoicesResponse>;
```

Comment:
I think we should just remove this function from the interface - it needs to take some params and we don't know what they are yet. We can add it later once we added the necessary backend support.

Reply:
Should we also remove GetInvoicesParams and GetInvoicesResponse types?

Reply:
yeah, let's remove all the code related to `getInvoices()` and we'll add it all back when we're ready.